packages/models/src/rooms.ts

```diff
@@ -98,6 +98,7 @@ export class Rooms {
       name: users.column("name"),
       image: users.column("image"),
       role: users.column("role"),
+      gender: users.column("gender"),
```

Comment:
It is great to see you making changes to the server if needed! This way you know how to unblock yourself instead of relaying on other team member. This crucial for your success as a team member!